packages/webpack/src/plugin.js

```diff
@@ -102,6 +102,18 @@ class LavaMoatPlugin {
     })
 
     const options = this.options
+
+    diag.run(2, () => {
+      // Log stack traces for all errors on higher verbosity because webpack won't
+      compiler.hooks.done.tap(PLUGIN_NAME, (stats) => {
+        if (stats.hasErrors()) {
+          stats.compilation.errors.forEach((error) => {
+            console.error(error.stack || error)
```

Comment:
```suggestion
            console.error(error)
```

Maybe better to let Node.js decide how to display it?